src/cpp/hu.bme.mit.inf.modes3.ESP32-sensor/railsensor_3/ESP32-sensor/lib/MQTT_JSON/MQTT_JSON.cpp

```diff
@@ -0,0 +1,123 @@
+#include <MQTT_JSON.hpp>
+
+void callback(char *topic, byte *payload, unsigned int length)
+{/*
+  StaticJsonBuffer<200> json;
+  JsonObject &root = json.parseObject((char *)payload);
+  if (!strcmp(type, "Ping"))
+  {
+    //PongSend();
+  }*/
+}
+
+int MQTT_JSON::getTime()
+{
+  return 100;
+}
+
+MQTT_JSON::MQTT_JSON(WiFiClient *wifi) :client(*wifi),
+                                        maxSize(800){}
+
+
+void MQTT_JSON::ConnCheck()
+{/*
+  if (WiFi.status() != WL_CONNECTED)
+  {
+    Serial.println("WiFi connection lost.");
+    //WifiConnect();
+  }
+  if (!client.connected())
+  {
+    Serial.println("MQTT connection lost.");
+    MQTTConnect();
+  }*/
+}
+
+void MQTT_JSON::PongSend()
+{
+  Serial.println("PongSend");
```

Comment:
Instead of repetitively copying and pasting the same code for every `MQTT_JSON::*Send()` method, you could extract the common lines into a function and invoke that function with parameters in which they differ. In this way, changing one line of code does not mean it has to be changed in N places (N = number of implementations of `*Send()` function).